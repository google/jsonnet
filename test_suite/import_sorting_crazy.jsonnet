local z = import 'zzzz.jsonnet'; /* zzz
zzz */ local y = import 'y.libsonnet'; /* aaaaa */ local x = import 'x.libsonnet';
/* w1 */ /* w2 */ local w = import 'w.jsonnet'; /* w3 */ /* w4 */
local u = import 'u.jsonnet';               local v = import 'v.jsonnet';
local t = import 't.jsonnet';local s=import 's.jsonnet';local r=import 'r.jsonnet';

local b = import "b.jsonnet";
/* foo */

/* bar */
local a = import "a.jsonnet";

true
