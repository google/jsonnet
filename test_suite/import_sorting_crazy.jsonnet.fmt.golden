local z = import 'zzzz.jsonnet';
/* zzz
zzz */
local x = import 'x.libsonnet';
local y = import 'y.libsonnet'; /* aaaaa */
/* w1 */ /* w2 */
local r = import 'r.jsonnet';
local s = import 's.jsonnet';
local t = import 't.jsonnet';
local u = import 'u.jsonnet';
local v = import 'v.jsonnet';
local w = import 'w.jsonnet'; /* w3 */ /* w4 */

local b = import "b.jsonnet";
/* foo */

/* bar */
local a = import "a.jsonnet";

true
